server/src/test/java/org/elasticsearch/action/admin/cluster/settings/SettingsUpdaterTests.java

```diff
@@ -57,14 +57,9 @@ public void testUpdateSetting() {
             .transientSettings(Settings.builder().put(BalancedShardsAllocator.INDEX_BALANCE_FACTOR_SETTING.getKey(), 3.5)
                 .put(BalancedShardsAllocator.SHARD_BALANCE_FACTOR_SETTING.getKey(), 4.5).build());
         ClusterState build = builder.metaData(metaData).build();
-        ClusterState clusterState = updater.updateSettings(build,
-            Settings.builder()
-                .put(BalancedShardsAllocator.INDEX_BALANCE_FACTOR_SETTING.getKey(), 0.5)
-                .build(),
-            Settings.builder()
-                .put(BalancedShardsAllocator.INDEX_BALANCE_FACTOR_SETTING.getKey(), 0.4)
-                .build(),
-            logger);
+        ClusterState clusterState = updater.updateSettings(build, Settings.builder()
```

Comment:
The old way was more readable I think.

Reply:
These were from merging in master :-/ I'll fix